provider/lxd/credentials.go

```diff
@@ -130,6 +139,12 @@ func (p environProviderCredentials) RegisterCredentials() (map[string]*cloud.Clo
 				lxdnames.DefaultCloud: *localCertCredential,
 			},
 		},
+		lxdnames.DefaultCloudAltName: {
```

Comment:
why register both, rather than just registering the one that matches?

CloudCredentials{
 credentialName:
}

Reply:
Sure, makes sense.